extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -1129,7 +1130,14 @@ private void setupUnixDomainSocketHttpServer(HttpServer httpServer, HttpServerOp
                         startFuture.complete(null);
                     }
                 } else {
-                    startFuture.fail(event.cause());
+                    if (event.cause() instanceof IllegalArgumentException) {
+                        startFuture.fail(new IllegalArgumentException(
+                                String.format(
+                                        "Unable to bind to Unix domain socket. Consider adding the 'io.netty:%s' dependency. See the Quarkus Vert.x reference guide for more details.",
```

Comment:
What's the URL for the Quarkus Vert.x reference guide?

Reply:
I didn't want to add that because that could change in the future